README.md

```diff
@@ -3,116 +3,104 @@ Universal neural network aimbot for all games with custom training mode
-2. Insert for enter in training mode (u can change it)
+#### Default settings (optional):
+* 320 for search range X and Y values
+* MiddleMouse to shoot
+* True for SimpleRCS
+* Insert to enable/disable Training Mode
+* Home to take a screenshot in Training Mode
+* Numpad9 to change screeshot mode (centered/following)
+* 0.1 for SmoothAim value
+* True to show real-time Info on your settings
+* False to aim at the head
 
-**In the config.json file, you can change the range resolution for object recognition, enable or disable SimpleRCS, specify which game we will use and specify the buttons for attack and training.**
+**In the config.json file, you can change the resolution range for object recognition, enable/disable SimpleRCS, specify which game you will use, and specify the button to attack and enable/disable Training Mode.**
 
-Requirements:
----
-1. Install the latest Nvidia driver for your graphic device
-2. Install [Nvidia CUDA Toolkit 10.0](https://developer.nvidia.com/cuda-toolkit-archive)(**!!!NOT 10.1!!!**) (must be installed add a hardware driver for cuda support)
+#### Training the NN:
+When you start the game for the first time, you will be asked to start training, this step is not optional (unless you already have a pre-trained NN for the game).
 
-If you have something missing, the program will tell you about it.
+During training, you must select the target object using PageUp and PageDown, take multiple screenshots of the target objects in the blue frame by pressing **HOME**(files should be created at darknet/data/img/), and once you're done, press the **END** button.
 
-How to train NN
----
-When you start the game for the first time, you will be immediately asked to start training. You can not refuse :)
-
-During a workout, you must select an object using special buttons(PageUP and PageDOWN), take screenshots of objects in the frame by pressing **HOME**(files should be created at darknet/data/img/), and after the procedure done, click on the **END** button.
-
-The training process for neural network begins.
+The training process for the neural network begins.
 
 After you are tired of waiting or you have reached the peak on the chart (description below), then press CTRL + C and confirm the action by writing Y and pressing enter in the window form(with a lot of text).
 
-After that, you definitely need to open the game again. Then the game is started, write to the console "done".
-Files should be copied into the trainfiles folder(automatically, but if it not then chekc files in the darknet(folder and subfolders) like a GAME.names(data folder) GAME.cfg(darknet folder) GAME_last.weights(backup folder, file shood be renamed to GAME.weights for program starts)
-
-How to train NN if you already trained it
----
-Press Insert button and u will come in training mode. If u press Insert again u will leave training mode.
+After that, you'll need to open the game again. When the game is started, write to the console `done`.
```

Comment:
After that, you will need to open the game again. When the game is started, write to the console `done`.